opm/parser/eclipse/EclipseState/IOConfig/IOConfig.hpp

```diff
@@ -161,6 +161,10 @@ namespace Opm {
         const std::string& getBaseName() const;
         void setBaseName(std::string baseName);
 
+        /// Return a string consisting of outputpath and basename;
+        /// i.e. /path/to/sim/CASE
+        std::string getOutputBase( ) const;
```

Comment:
The naming `baseName` and `outputBase` confuses me.  Could it be possible to rename `outputBase` to `fullPath` or `fullBasePath` or something?

Why does `getOutputBase` need to contain _output_ while `getBaseName` doesn't?